api/v1alpha1/nodeobservabilityrun_types.go

```diff
@@ -54,6 +54,7 @@ const (
 	CrioKubeletProfile NodeObservabilityRunType = "crioKubeletProfile"
 	EbpfProfile        NodeObservabilityRunType = "eBPFprofile"
 	E2ETest            NodeObservabilityRunType = "e2e-test"
+	Disabled           NodeObservabilityRunType = "disabled"
```

Comment:
I'm wondering if we could use another word . 
a NodeObservabilityRun of type `disabled` might be confusing to the end user.
Propositions: plain, standard, default

Reply:
Included Disabled to avoid creating NodeObservabilityMachineConfig during each reconciling once after 30s window of profiling was completed, but RestoreMCOStateAfterRun serves way for retaining the MC related changes, hence removed it.